index.js

```diff
@@ -1,13 +1,82 @@
-async function getWeather(city = null) {
+const APP_KEY = 'ffa2b0f503f54446b0d77a7cc99187f0';
+/* eslint-disable consistent-return,prefer-destructuring */
+async function getWeather(city = 'Delhi') {
   try {
     const data = await fetch(
-      `https://api.openweathermap.org/data/2.5/weather?q=${city}&appid=${APP_KEY}`
+      `https://api.weatherbit.io/v2.0/current?city=${city}&key=${APP_KEY}`,
     );
-    const weather = await data.json();
-    console.log(weather);
+    if (data.status === 200) {
+      const weather = await data.json();
+
+      return weather.data[0];
+    }
   } catch (err) {
-    console.log(err);
+    return 'Error';
+  }
+}
+
```

Comment:
we need to make different requests for the Celsius data and the Fahrenheit data.
![image](https://user-images.githubusercontent.com/31897434/84801477-245bcb80-aff7-11ea-9685-e28a5bb373b1.png)
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622133/